test/utils/download_and_install_keptn_cli.sh

```diff
@@ -7,4 +7,5 @@ wget "https://storage.googleapis.com/keptn-cli/${KEPTN_CLI_VERSION}/keptn-linux.
 unzip keptn-linux.zip
 
 sudo mv keptn /usr/local/bin/keptn
-
+echo "Keptn CLI installed in version:"
+keptn version
```

Comment:
I assume that here we will get the output (see `Keptn cluster version`): 
```
> keptn version

Keptn CLI version: 0.8.0
Keptn cluster version: Error connecting to server: Head http://xx.xx.xx.xx/api: context deadline exceeded
Possible reasons:
* The Keptn API server is currently not available. Check if your Kubernetes cluster is available.
* Your Keptn CLI points to the wrong API server (verify using 'keptn status')

Daily version check is enabled.
Keptn will collect statistical data and will notify about new versions and security patches for Keptn. Details can be found at: https://keptn.sh/docs/0.7.x/reference/version_check
---------------------------------------------------
* To disable the daily version check, please execute:
 - keptn set config AutomaticVersionCheck false
```

But this is okay since there is no Keptn installed yet.

Reply:
I have moved the `keptn version` command to execute it after the installation of Keptn in the cluster